### ISSUE TITLE
Cannot see BBC logo on high contrast mode in Windows IE

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#860]
**Describe the bug**
When viewing the page in IE on PC with 'High Contrast White' mode enabled I cannot see the BBC News logo.

It maybe useful to watch this talk for implementation advice https://www.youtube.com/watch?v=CJltSqksCn0 and also read this by Jenny https://confluence.dev.bbc.co.uk/display/~jenny.robertson@bbc.co.uk/Creating+content+for+high+contrast+users

Looking at this quickly, we probably want an inline fill of black (users can then see the SVG on a white background when CSS is disabled), then some CSS to make the fill white when the user has CSS (so this can be seen on the red background and match the design), and also some CSS using a media query for when windows high contrast mode is active to set the fill to 'windowText' so the SVG can be seen in whatever colour theme the user has set (or CSS 'fill: currentColor' without a media query may work - see youtube video at about 30 minutes in).

I would suggest that we decide on an approach for the ZoomText bug before addressing this bug as it may have a knock on effect - And if using an inline fill colour of black while addressing the high contrast bug that we should also make the logo a reasonable size when CSS is disabled - currently it fills the screen.

**To Reproduce**
View an articles page with high-contrast mode on in IE on Windows.

**Expected behavior**
Should be able to see the BBC News logo.

**Desktop (please complete the following information):**

- OS: Windows
- Browser IE
- High-contrast mode

**Comment by BogdanDogaru** [comment]
@greenc05 would you be able to add a screenshot for this? It would greatly help us.

**Comment by greenc05** [comment]
Sure @BogdanDogaru Here is a screen shot of Windows High Contrast Mode 1 where you can see the BBC News logo and the Windows High Contrast Mode White where you cannot see the BBC News logo

![high-contrast-mode-1](https://user-images.githubusercontent.com/1061401/47991040-edb29500-e0e0-11e8-85fd-388ff6361e8a.png)

![high-contrast-mode-white](https://user-images.githubusercontent.com/1061401/47991054-f4410c80-e0e0-11e8-8ef2-2bb0687020f4.png)

**Comment by ChrisBAshton** [comment]
Related to #378.

**Comment by dr3** [comment]
![image](https://user-images.githubusercontent.com/11341355/50636925-da407080-0f4f-11e9-83d3-21b240091d02.png)
